README.md

```diff
@@ -16,38 +16,18 @@ Based on a review of the Education themed content, this algorithm performed sign
 First, you'll need to install some dependencies. It's a python thing, so:
 
 ```
-$ git clone git@github.com:alphagov/ordering-documents-by-inverse-similarity.git
-$ cd ordering-documents-by-inverse-similarity
-$ virtualenv .venv
-$ source .venv/bin/activate
-$ pip install -r requirements.txt
+$ git clone git@github.com:alphagov/govuk-inverse-similarity.git
+$ govuk-inverse-similarity
```

Comment:
This line isn't needed - the file doesn't exist so I'm not sure if this was intended on being some kind of setup script?

Reply:
Ah, I've missed the `cd` off of the start. I'll update. Good spot :)